[2022-10-23 01:43:07,319] [util.py:209]               [INFO]     |====================================================================================================|
[2022-10-23 01:43:07,320] [util.py:216]               [INFO]     |                                 Popular TV Collection in TV Shows                                  |
[2022-10-23 01:43:07,320] [util.py:220]               [INFO]     |====================================================================================================|
[2022-10-23 01:43:07,320] [plex_meta_manager.py:750]  [INFO]     |                                                                                                    |
[2022-10-23 01:43:07,321] [util.py:216]               [INFO]     |================================= Validating Popular TV Attributes =================================|
[2022-10-23 01:43:07,321] [builder.py:330]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:07,321] [builder.py:331]            [DEBUG]    | Validating Method: sync_mode                                                                       |
[2022-10-23 01:43:07,322] [builder.py:335]            [DEBUG]    | Value: sync                                                                                        |
[2022-10-23 01:43:07,322] [builder.py:389]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:07,322] [builder.py:390]            [DEBUG]    | Validating Method: smart_label                                                                     |
[2022-10-23 01:43:07,322] [builder.py:395]            [DEBUG]    | Value: originally_available.desc                                                                   |
[2022-10-23 01:43:07,323] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:07,323] [builder.py:473]            [DEBUG]    | Validating Method: sort_title                                                                      |
[2022-10-23 01:43:07,323] [builder.py:474]            [DEBUG]    | Value: +1b                                                                                         |
[2022-10-23 01:43:07,324] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:07,324] [builder.py:473]            [DEBUG]    | Validating Method: imdb_list                                                                       |
[2022-10-23 01:43:07,324] [builder.py:474]            [DEBUG]    | Value: ordereddict([('url', 'https://www.imdb.com/search/title/?title_type=tv_series,tv_miniseries'), ('limit', 40)]) |
[2022-10-23 01:43:08,130] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,130] [builder.py:473]            [DEBUG]    | Validating Method: summary                                                                         |
[2022-10-23 01:43:08,130] [builder.py:474]            [DEBUG]    | Value: Popular Shows across the internet                                                           |
[2022-10-23 01:43:08,131] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,131] [builder.py:473]            [DEBUG]    | Validating Method: url_poster                                                                      |
[2022-10-23 01:43:08,131] [builder.py:474]            [DEBUG]    | Value: https://i.imgur.com/s90PSic.png                                                             |
[2022-10-23 01:43:08,131] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,132] [builder.py:473]            [DEBUG]    | Validating Method: url_background                                                                  |
[2022-10-23 01:43:08,132] [builder.py:474]            [DEBUG]    | Value: https://wallpaper.dog/large/20509069.jpg                                                    |
[2022-10-23 01:43:08,132] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,133] [builder.py:473]            [DEBUG]    | Validating Method: visible_home                                                                    |
[2022-10-23 01:43:08,133] [builder.py:474]            [DEBUG]    | Value: True                                                                                        |
[2022-10-23 01:43:08,133] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,134] [builder.py:473]            [DEBUG]    | Validating Method: visible_shared                                                                  |
[2022-10-23 01:43:08,134] [builder.py:474]            [DEBUG]    | Value: True                                                                                        |
[2022-10-23 01:43:08,134] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,134] [builder.py:473]            [DEBUG]    | Validating Method: visible_library                                                                 |
[2022-10-23 01:43:08,135] [builder.py:474]            [DEBUG]    | Value: True                                                                                        |
[2022-10-23 01:43:08,135] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,135] [builder.py:473]            [DEBUG]    | Validating Method: sonarr_add_missing                                                              |
[2022-10-23 01:43:08,135] [builder.py:474]            [DEBUG]    | Value: True                                                                                        |
[2022-10-23 01:43:08,178] [builder.py:619]            [INFO]     |                                                                                                    |
[2022-10-23 01:43:08,178] [builder.py:620]            [INFO]     | Validation Successful                                                                              |
[2022-10-23 01:43:08,180] [plex_meta_manager.py:758]  [INFO]     |                                                                                                    |
[2022-10-23 01:43:08,180] [util.py:216]               [INFO]     |================================== Running Popular TV Collection ===================================|
[2022-10-23 01:43:08,180] [plex_meta_manager.py:773]  [INFO]     |                                                                                                    |
[2022-10-23 01:43:08,180] [plex_meta_manager.py:774]  [INFO]     | Sync Mode: sync                                                                                    |
[2022-10-23 01:43:08,181] [builder.py:1275]           [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:08,181] [builder.py:1276]           [DEBUG]    | Builder: imdb_list: {'url': 'https://www.imdb.com/search/title/?title_type=tv_series,tv_miniseries', 'limit': 40} |
[2022-10-23 01:43:08,181] [builder.py:1277]           [INFO]     |                                                                                                    |
[2022-10-23 01:43:08,182] [imdb.py:158]               [INFO]     | Processing IMDb List: 40 Items at https://www.imdb.com/search/title/?title_type=tv_series,tv_miniseries |
[2022-10-23 01:43:11,409] [imdb.py:127]               [DEBUG]    | 40 IMDb IDs Found: ['tt11198330', 'tt7631058', 'tt13444912', 'tt13207736', 'tt10857160', 'tt14852808', 'tt9253284', 'tt1520211', 'tt0944947', 'tt13443470', 'tt7120662', 'tt2861424', 'tt1844624', 'tt5834204', 'tt4574334', 'tt0903747', 'tt14921986', 'tt3032476', 'tt0413573', 'tt13720112', 'tt14452776', 'tt4786824', 'tt12661338', 'tt13406094', 'tt7949218', 'tt13616990', 'tt1190634', 'tt4236770', 'tt14923244', 'tt15469618', 'tt0429087', 'tt5645432', 'tt16098700', 'tt7221388', 'tt15574312', 'tt0460681', 'tt2442560', 'tt12590266', 'tt2741602', 'tt0364845'] |
[2022-10-23 01:43:11,409] [builder.py:1282]           [DEBUG]    |                                                                                                    |
[2022-10-23 01:43:11,410] [builder.py:1283]           [DEBUG]    | 40 IDs Found: [('tt11198330', 'imdb'), ('tt7631058', 'imdb'), ('tt13444912', 'imdb'), ('tt13207736', 'imdb'), ('tt10857160', 'imdb'), ('tt14852808', 'imdb'), ('tt9253284', 'imdb'), ('tt1520211', 'imdb'), ('tt0944947', 'imdb'), ('tt13443470', 'imdb'), ('tt7120662', 'imdb'), ('tt2861424', 'imdb'), ('tt1844624', 'imdb'), ('tt5834204', 'imdb'), ('tt4574334', 'imdb'), ('tt0903747', 'imdb'), ('tt14921986', 'imdb'), ('tt3032476', 'imdb'), ('tt0413573', 'imdb'), ('tt13720112', 'imdb'), ('tt14452776', 'imdb'), ('tt4786824', 'imdb'), ('tt12661338', 'imdb'), ('tt13406094', 'imdb'), ('tt7949218', 'imdb'), ('tt13616990', 'imdb'), ('tt1190634', 'imdb'), ('tt4236770', 'imdb'), ('tt14923244', 'imdb'), ('tt15469618', 'imdb'), ('tt0429087', 'imdb'), ('tt5645432', 'imdb'), ('tt16098700', 'imdb'), ('tt7221388', 'imdb'), ('tt15574312', 'imdb'), ('tt0460681', 'imdb'), ('tt2442560', 'imdb'), ('tt12590266', 'imdb'), ('tt2741602', 'imdb'), ('tt0364845', 'imdb')] |
[2022-10-23 01:43:11,410] [builder.py:1284]           [DEBUG]    |                                                                                                    |
[2022-10-23 01:44:01,499] [util.py:162]               [DEBUG]    | Traceback (most recent call last):                                                                 |
[2022-10-23 01:44:01,499] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\requests\models.py", line 910, in json |
[2022-10-23 01:44:01,500] [util.py:162]               [DEBUG]    |     return complexjson.loads(self.text, **kwargs)                                                  |
[2022-10-23 01:44:01,500] [util.py:162]               [DEBUG]    |   File "C:\Users\Jack\AppData\Local\Programs\Python\Python39\lib\json\__init__.py", line 346, in loads |
[2022-10-23 01:44:01,500] [util.py:162]               [DEBUG]    |     return _default_decoder.decode(s)                                                              |
[2022-10-23 01:44:01,500] [util.py:162]               [DEBUG]    |   File "C:\Users\Jack\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 337, in decode |
[2022-10-23 01:44:01,501] [util.py:162]               [DEBUG]    |     obj, end = self.raw_decode(s, idx=_w(s, 0).end())                                              |
[2022-10-23 01:44:01,501] [util.py:162]               [DEBUG]    |   File "C:\Users\Jack\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 355, in raw_decode |
[2022-10-23 01:44:01,501] [util.py:162]               [DEBUG]    |     raise JSONDecodeError("Expecting value", s, err.value) from None                               |
[2022-10-23 01:44:01,501] [util.py:162]               [DEBUG]    | json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)                            |
[2022-10-23 01:44:01,502] [util.py:162]               [DEBUG]    |                                                                                                    |
[2022-10-23 01:44:01,502] [util.py:162]               [DEBUG]    | During handling of the above exception, another exception occurred:                                |
[2022-10-23 01:44:01,502] [util.py:162]               [DEBUG]    |                                                                                                    |
[2022-10-23 01:44:01,502] [util.py:162]               [DEBUG]    | Traceback (most recent call last):                                                                 |
[2022-10-23 01:44:01,503] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\plex_meta_manager.py", line 783, in run_collection         |
[2022-10-23 01:44:01,503] [util.py:162]               [DEBUG]    |     builder.find_rating_keys()                                                                     |
[2022-10-23 01:44:01,503] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\builder.py", line 1379, in find_rating_keys        |
[2022-10-23 01:44:01,503] [util.py:162]               [DEBUG]    |     tvdb_id = self.config.Convert.tmdb_to_tvdb(tmdb_id, fail=True)                                 |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\convert.py", line 149, in tmdb_to_tvdb             |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |     tvdb_id = self.config.TMDb.convert_from(tmdb_id, "tvdb_id", False)                             |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\retrying.py", line 49, in wrapped_f |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |     return Retrying(*dargs, **dkw).call(f, *args, **kw)                                            |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\retrying.py", line 212, in call |
[2022-10-23 01:44:01,504] [util.py:162]               [DEBUG]    |     raise attempt.get()                                                                            |
[2022-10-23 01:44:01,505] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\retrying.py", line 247, in get  |
[2022-10-23 01:44:01,505] [util.py:162]               [DEBUG]    |     six.reraise(self.value[0], self.value[1], self.value[2])                                       |
[2022-10-23 01:44:01,505] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\six.py", line 719, in reraise   |
[2022-10-23 01:44:01,505] [util.py:162]               [DEBUG]    |     raise value                                                                                    |
[2022-10-23 01:44:01,505] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\retrying.py", line 200, in call |
[2022-10-23 01:44:01,506] [util.py:162]               [DEBUG]    |     attempt = Attempt(fn(*args, **kwargs), attempt_number, False)                                  |
[2022-10-23 01:44:01,506] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\tmdb.py", line 91, in convert_from                 |
[2022-10-23 01:44:01,506] [util.py:162]               [DEBUG]    |     id_to_return = self.Movie.external_ids(tmdb_id)[convert_to] if is_movie else self.TV.external_ids(tmdb_id)[convert_to] |
[2022-10-23 01:44:01,506] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\tmdbv3api\objs\tv.py", line 149, in external_ids |
[2022-10-23 01:44:01,507] [util.py:162]               [DEBUG]    |     self._call(self._urls["external_ids"] % (str(id)), ""), None                                   |
[2022-10-23 01:44:01,507] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\tmdbv3api\tmdb.py", line 156, in _call |
[2022-10-23 01:44:01,507] [util.py:162]               [DEBUG]    |     json = req.json()                                                                              |
[2022-10-23 01:44:01,507] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\pmm-venv\lib\site-packages\requests\models.py", line 917, in json |
[2022-10-23 01:44:01,507] [util.py:162]               [DEBUG]    |     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)                                             |
[2022-10-23 01:44:01,508] [util.py:162]               [DEBUG]    | requests.exceptions.JSONDecodeError: [Errno Expecting value] : 0                                   |
[2022-10-23 01:44:01,508] [util.py:162]               [DEBUG]    |                                                                                                    |
[2022-10-23 01:44:01,508] [plex_meta_manager.py:877]  [ERROR]    | Unknown Error: [Errno Expecting value] : 0                                                         |
[2022-10-23 01:44:01,508] [plex_meta_manager.py:880]  [INFO]     |                                                                                                    |
[2022-10-23 01:44:01,509] [util.py:209]               [INFO]     |====================================================================================================|
[2022-10-23 01:44:01,509] [util.py:216]               [INFO]     |                                   Finished Popular TV Collection                                   |
[2022-10-23 01:44:01,509] [util.py:216]               [INFO]     |                                    Collection Run Time: 0:00:54                                    |
[2022-10-23 01:44:01,510] [util.py:220]               [INFO]     |====================================================================================================|
